truffleruby/src/main/ruby/core/enumerable.rb

```diff
@@ -36,11 +36,11 @@ def chunk(&original_block)
     ::Enumerator.new do |yielder|
       previous = nil
       accumulate = []
-      block = if initial_state.nil?
-        original_block
+      if initial_state.nil?
+        block = original_block
       else
         duplicated_initial_state = initial_state.dup
-        Proc.new{ |val| original_block.yield(val, duplicated_initial_state)}
+        block = Proc.new{ |val| original_block.yield(val, duplicated_initial_state)}
```

Comment:
Rubocop wants to have `if`/`else`/`end` on the same line like so:
```ruby
block = if initial_state.nil?
          original_block
        else
          duplicated_initial_state = initial_state.dup
          Proc.new{ |val| original_block.yield(val, duplicated_initial_state)}
        end
```
I don't particularly like this style (indent depending on variable name length) and in this situation I find the duplicated assignment clearer. Sometimes it allows to remove one branch which does just `var = var` which is kind of nice.

But in any case, both are accepted by Rubocop, the `if` not aligned with `else`/`end` is what triggers the error.

Interestingly enough for `case` it can be configured to not require the `case` to be aligned but just require that the `when` and the `end` are aligned. That's what I chose for `case` as it is more flexible.